api/src/test/java/org/openmrs/serialization/SimpleXStreamSerializerTest.java

```diff
@@ -91,4 +90,18 @@ public void deserialize_shouldDeserializeStringToClassInstance() throws Serializ
 		Assert.assertTrue(newMap.get(30).equals("bar"));
 		
 	}
+	
+	/**
+	 * @see SimpleXStreamSerializer#deserialize(String,Class)
+	 * @verifies not deserialize dynamic proxies
+	 */
+	@Test
+	public void deserialize_shouldNotDeserializeDynamicProxies() throws Exception {
+		String serialized = "<dynamic-proxy>" + "<interface>org.openmrs.OpenmrsObject</interface>"
```

Comment:
Could you please change the content here to be something that is _not_ a blueprint for an attack string that could work on older OpenMRS versions?